dev-libs/libaddressinput/libaddressinput-0~git.recipe

```diff
@@ -0,0 +1,87 @@
+SUMMARY="Google’s postal address library that powering Android and Chromium"
+DESCRIPTION="libaddressinput can provide information about \
+what input fields are required for a correct address input form for \
+any country in the world and can validate an address to highlight \
+input errors like missing required fields or invalid values."
+HOMEPAGE="https://github.com/googlei18n/libaddressinput"
+COPYRIGHT="2014 Google Inc."
+LICENSE="Apache v2"
+REVISION="1"
+srcGitRev="a575c63de328d0c72237c55d1a05e191c245122a"
+SOURCE_URI="https://github.com/googlei18n/libaddressinput/archive/$srcGitRev.tar.gz"
+CHECKSUM_SHA256="f92b38a0291372aa97306626f2941485bd1c00bd3cc7e242a1cb9fcd50b62212"
+SOURCE_FILENAME="libaddressinput-$portVersion-$srcGitRev.tar.gz"
+SOURCE_DIR="libaddressinput-$srcGitRev"
+googletestVersion="1.8.0"
+SOURCE_URI_2="https://github.com/google/googletest/archive/release-$googletestVersion.tar.gz"
+CHECKSUM_SHA256_2="58a6f4277ca2bc8565222b3bbd58a177609e9c488e8a72649359ba51450db7d8"
+SOURCE_FILENAME_2="gtest-$googletestVersion.tar.gz"
+srcGitRev3="b033674"
+SOURCE_URI_3="https://chromium.googlesource.com/chromium/src/tools/grit/+archive/$srcGitRev3.tar.gz"
+CHECKSUM_SHA256_3="d5a0519983b494f7c5cec5c6e109f52ba33d4f3c47ef52c033b3dcab0cdb7349"
+SOURCE_FILENAME_3="grit-$srcGitRev3.tar.gz"
+
+ARCHITECTURES="!x86_gcc2 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	libaddressinput$secondaryArchSuffix = $portVersion
+	lib:libaddressinput$secondaryArchSuffix = portVersion
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libre2$secondaryArchSuffix
+	"
+
+PROVIDES_devel="
+	libaddressinput${secondaryArchSuffix}_devel
+	devel:libaddressinput$secondaryArchSuffix
+	"
+REQUIRES_devel="
+	libaddressinput$secondaryArchSuffix == $portVersion base
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:rapidjson <= 1.1.0
+	devel:libgtest$secondaryArchSuffix
```

Comment:
Since you're using a bundled `gtest`

```diff
-	devel:libgtest$secondaryArchSuffix

Reply:
Bundled `gtest` is only for providing source code. But libaddressinput requires both source code and library.

Reply:
Then you should [modify gtest recipe to bundle it's sources][0] and use that instead. Because we need to keep gtest and the bundled gtest synchronized, this is the best way (that I could think of) to solve this.

[0]: https://github.com/haikuports/haikuports/pull/2107#discussion_r161416977